backend/uclapi/requirements.txt

```
@@ -27,7 +27,7 @@ psycopg2-binary==2.8.6
 python-dateutil==2.8.1
 pymsteams==0.1.14
 pytz==2020.4
-raven==6.10.0
+sentry-sdk==0.19.5
```

```suggestion
sentry-sdk==0.20.1
```
Benefits of not reviewing... although test that it doesn't break anything :P